Include/object.h

```diff
@@ -790,15 +789,25 @@ static inline void _Py_ForgetReference(PyObject *op)
 {
     _Py_INC_TPFREES(op);
 }
+#endif /* !Py_TRACE_REFS */
+
 
-#ifdef Py_LIMITED_API
 PyAPI_FUNC(void) _Py_Dealloc(PyObject *);
+
+#ifndef Py_LIMITED_API
+static inline void _Py_Dealloc_inline(PyObject *op)
```

Comment:
Shouldn't there be a line break after "void"?

Reply:
I prefer to keep header files short, and I chose to use a similar style for static inline than for macros: #define name(x) => static inline void name(PyObject *x) { .... }. If you want to change it, all other static inline function should be used. I would prefer to keep it on a single line (for the specific case of static inline functions in header files).

Reply:
This just doesn't confirm PEP 7. It looks fine for functions with short signatures, but becomes awkward if the function has 4 parameters.

On other hand, the return time is allowed to be on the same name with the name in declarations. And inline functions are definitions and declarations at the same time. So it may be that this PEP 7 rule is not applicable here. I trust your feeling, if this will be a problem, we could change it in future.

Reply:
If you would to change the coding style, I would prefer to do it in a separated PR, since I already added multiple "static inline" function in header files with this style.

I don't have a strong opinion on exact coding style.